src/lib/pages/workflow-pending-activities.svelte

```diff
@@ -123,8 +123,9 @@
               <p>
                 {`${formatDate(details.lastHeartbeatTime)} (${formatDate(
                   details.lastHeartbeatTime,
-                  'relative',
+                  'local',
```

Comment:
We're currently only showing the relative time here, but I think this could also probably be updated to use `$timeFormat`.

![Screenshot 2023-08-16 at 10 04 31 AM](https://github.com/temporalio/ui/assets/15069288/5c9ab2bb-97bc-411c-98f3-c8800b77925a)